### DISCUSSION TITLE
Q: 评论系统可以正常评论，但是评论列表不展示 || Q: The comment system can comment normally, but the comment list does not display

### DISCUSSION THREAD

**Discussion by Huangshi-maker** [walinejs/waline#2007] [Q&A] [answered]
### Describe the bug | 问题描述

在使用hexo创建个人博客时，使用vercel部署评论系统，但是评论后内容刷新后不会展示评论列表，只存在数据库中，使用vercel访问的也不展示列表，这个具体怎么解决呢？

### Website URL | 问题网站

https://waline.icemmq.com.cn

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

When using hexo to create a personal blog, use vercel to deploy the comment system, but the comment list will not be displayed after the content is refreshed after the comment, it only exists in the database, and the list will not be displayed when using vercel to access.

### Website URL | Problem Website

https://waline.icemmq.com.cn

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by Huangshi-maker** [comment]
<img width="975" alt="image" src="https://github.com/walinejs/waline/assets/77331529/47b08398-51cd-4ce5-a636-9928cb6f0e5e">
是不是这个问题导致无法正常加载，应该怎么解决

**Comment by lizheming** [comment, marked as answer]
后台看下数据状态，是不是审核中

---
Check the data status in the background, is it under review

**Reply by Huangshi-maker** [reply]
<img width="506" alt="image" src="https://github.com/walinejs/waline/assets/77331529/15d9a109-b1e2-462c-92c4-8e9d24b1c094">
状态全部是waiting

**Reply by Huangshi-maker** [reply]
刚发现没有登录的话没有评论列表展示的，然后我登录之后可以看到评论列表，有没有什么办法不登录也能显示评论列表的


---
I just found that if I am not logged in, there is no comment list displayed, and then I can see the comment list after I log in. Is there any way to display the comment list without logging in?

**Reply by Huangshi-maker** [reply]
不好意思，我知道什么原因了，谢谢您！

---
Sorry, I know the reason, thank you!
